### PR TITLE
Add practice exercise `pop-count`

### LINKED ISSUE #615: Implement new pop-count
I'd like to port this exercise introduced in exercism/problem-specifications#2333.

### PR DESCRIPTION
**Pull request by BNAndras**
Will close #615

### CODE REVIEW

**Comment by BNAndras** [comment]
Oops, I had elm-review suggest pruning unneeded dependencies from elm.json and didn't think twice about it.

**Comment by ceddlyburge** [comment]
> Oops, I had elm-review suggest pruning unneeded dependencies from elm.json and didn't think twice about it.

Yeah, this is a bi of an annoyance. All the elm.json files must be the same, its a requirement of the test runner, which has to makle sure that it has all the dependencies in the elm.json files.

File: config.json
Comment (ceddlyburge) [inline config.json:583]: Are there any concepts this exercise practices, or that are required?

**Review by ceddlyburge — COMMENTED** [review]
This all looks good to me. I'll let Jie / Matthieu take a look as well, and you'll need to fix up the elm.json in order for the build to pass. I'll approve when it does. The tests can be flaky, so if that happens let me know and I'll re run them.
Thanks for the contribution!
Cheers, Cedd

File: config.json
Comment (BNAndras) [reply config.json:583]: I wasn't sure. Normally, I'd gravitate towards a `numbers` concept if available for both the practices or requires arrays. Since that's not here, perhaps `basics-2` for both since it touches upon numbers?

Comment (jiegillet) [reply config.json:583]: Yes, I think `basics-2` would be great for the practices fiels, and `basics-1`, basics-2 for the prerequisites. It's nice to have an exercise available so early in the tree.

**Review by jiegillet — COMMENTED** [review]
I'm good with this as well (once the things already discussed are settled), thank you for porting this.

I was wondering if we needed an analyzer comment to prevent people from using a built-in `popCount`, but there is no such function in `Bitwise` and I think `elm/bytes` is also pretty unrelated, so we should be good.

File: config.json
Comment (jiegillet) [inline config.json:10]: That's too many changes on the config, I can't be sure of what the actual change is. I'm assuming it got sorted somehow, but could you revert that?

Comment (BNAndras) [reply config.json:10]: I had to run configlet fmt to make the CI pass. There was a recent release for configlet so I think that’s the issue. I can open a separate PR for the config formatting. That’ll make it easier to see the actual changes.

Comment (jiegillet) [reply config.json:10]: Oh I see, thank you for pointing it out. Please leave it like that, I checked that it's only adding what it's supposed to.

But that's strange, when I run `bin/configlet fmt` is says everything is clear, but running `bin/configlet fmt --update` on main it changes this file. Is it the same for you? I should flag that.

**Review by jiegillet — APPROVED** [review]
Thank you for adding this one :)

**Pull request merged**
